src/dmd/statementsem.d

```diff
@@ -3737,7 +3743,8 @@ else
 
         auto blockexit = tfs._body.blockExit(sc.func, false);
 
-        if (!global.params.useExceptions)       // if not worrying about exceptions
+        // if not worrying about exceptions
+        if (!global.params.useExceptions || !ClassDeclaration.throwable)
```

Comment:
Elsewhere is written `useExceptions && throwable`, so I suggest rewriting this to be consistent:
```
if (!(global.params.useExceptions && ClassDeclaration.throwable))
```

Reply:
Done